exploration/demo/evaluate.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+import polars as pl
+
+from linmod.eval.proportions import mae
+
+data = (
+    pl.scan_csv(Path("../../data/metadata.csv"))
```

Comment:
`Path()` is redundant here. I think the "elegant" solution would be to have the root be known, so that you could say
```python
pl.scan_csv(root_path / "data" / "metadata.csv")
```